contrib/backup/functions

```diff
@@ -191,11 +207,12 @@ function restore_zammad () {
       ${ZAMMAD_DIR}/bin/rake db:create
     fi
 
-    echo "# Restoring PostgrSQL DB"
-    zcat ${BACKUP_DIR}/${RESTORE_DB_DATE}_zammad_db.${DB_FILE_EXT}.gz | su -c "psql -d ${DB_NAME}" zammad
+    echo "# Restoring PostgreSQL DB"
+    zcat < ${BACKUP_DIR}/${RESTORE_DB_DATE}_zammad_db.${DB_FILE_EXT}.gz \
+      | psql ${DB_HOST:+--host $DB_HOST} ${DB_PORT:+--port $DB_PORT} ${DB_USER:+--username $DB_USER} --dbname ${DB_NAME}
```

Comment:
This one does not include the password (I din't touch this code in #3352).

Reply:
Added in the current version.